src/helpers/configuration.sh

```diff
@@ -24,7 +24,7 @@ fi
 if [[ "$(is_git_town_configured)" == false ]]; then
   echo "Git Town hasn't been configured for this repository."
   echo "Please run 'git town config --setup'."
-  echo -n "Would you like to do that now? [y/n] "
+  echo -n "Would you like to do that now?"
```

Comment:
I think we're talking about different things. I had only moved the location of the Y/n in code, it would still show the user the same way. I mistakenly removed Y/n in the spec. Added that back.